packages/jss-plugin-syntax-nested/src/index.test.js

```diff
@@ -637,4 +637,32 @@ describe('jss-plugin-syntax-nested', () => {
       `)
     })
   })
+
+  describe('nest rules inside media query', () => {
+    let sheet
+
+    beforeEach(() => {
+      const localJss = create(settings).use(nested())
```

Comment:
it seems there is no ned to set up localJss instance, the one defined in the beginning has exact same setup, you can just use `jss` ...  we used a new one because it needed a different setup